src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -628,6 +628,12 @@
         - name: --bootstrap-container-registry-resource-id
           type: string
           short-summary: Configure container registry resource ID. Must use "Cache" as bootstrap artifact source.
+        - name: --if-match
+          type: string
+          short-summary: The ETag of the managed cluster. Omit this value to create a managed cluster.
```

Comment:
I don't love this description. We should talk about what happens if the etag doesn't match and what happens if it does.

On the second sentence, is: `This must not be specified when creating a new cluster` clearer than what you have?

Reply:
Updated